packages/page-staking/src/Query/Validator.tsx

```diff
@@ -12,18 +12,25 @@ import ChartPoints from './ChartPoints';
 import ChartPrefs from './ChartPrefs';
 import ChartRewards from './ChartRewards';
 import ChartStake from './ChartStake';
+import { useApi } from '@polkadot/react-hooks';
 
 function Validator ({ className = '', validatorId }: Props): React.ReactElement<Props> {
+  const { api } = useApi();
+  const isAlephChain = useMemo(() => {
+    return api.runtimeChain.toString().includes('Aleph Zero');
+  }, [api]
+  );
+
   return (
     <Columar className={className}>
-      <Columar.Column>
-        <ChartPoints validatorId={validatorId} />
-        <ChartRewards validatorId={validatorId} />
-      </Columar.Column>
       <Columar.Column>
         <ChartStake validatorId={validatorId} />
         <ChartPrefs validatorId={validatorId} />
       </Columar.Column>
+      <Columar.Column>
```

Comment:
whye the change in the order of Columar.Column?

Reply:
I guess it's to avoid column switching place depending on selected chain